.github/workflows/test.yml

```diff
@@ -5,23 +5,27 @@ on:
 
 jobs:
   test:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-20.04  # ubuntu-latest
 
     steps:
     - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.0
-    - name: Set up Python 3.8
+    - name: Set up Python 3.11
```

Comment:
>Also a bit confused why you pick 3.11 here and 3.10 elsewhere.

This is probably just because I had local troubles with Ansible being dependent on a security package that I only get working under 3.10. Otherwise 3.11 should be the standard.